docs/Using-Fleet/MDM-disk-encryption.md

```diff
@@ -93,6 +100,8 @@ How to view the disk encryption key:
 
 2. On the **Host details** page, select **Actions > Show disk encryption key**.
 
+> Whenever a disk encryption key is viewed, an activity will be logged.
```

Comment:
```suggestion
> Whenever a disk encryption key is viewed, an activity will be logged. To view activity in the Fleet UI, click on the Fleet icon in the top navigation bar and make sure **All teams** is selected in the teams dropdown.
```

Reply:
Added instructions for viewing the activity.

Reply:
Gosh that dashboard is crazy.

Reply:
Yeah it needs some love